OpenRA.Mods.Common/Traits/Render/RenderJammerCircle.cs

```diff
@@ -18,8 +18,20 @@
 namespace OpenRA.Mods.Common.Traits
 {
 	// TODO: remove all the Render*Circle duplication
-	class RenderJammerCircleInfo : TraitInfo<RenderJammerCircle>, IPlaceBuildingDecorationInfo
+	class RenderJammerCircleInfo : TraitInfo, IPlaceBuildingDecorationInfo
 	{
+		[Desc("Range circle outline contrast color")]
+		public readonly Color CircleColor = Color.FromArgb(128, Color.Red);
```

Comment:
These shouldn't start with `Circle`, to match the other dedicated `Render*Circle` traits.